src/test/suite/dbtProject.test.ts

```
@@ -0,0 +1,413 @@
+
+class MockDBTProjectLog extends DBTProjectLog {
+  constructor() {
+    const mockConfigEvent = new EventEmitter<ProjectConfigChangedEvent>();
+    super(mockConfigEvent.event);
+  }
+}
+
+class MockValidationProvider extends ValidationProvider {
+  constructor(altimate: any) {
+    super(altimate);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)